examples/minimal-appdir/src/app/page.tsx

```diff
@@ -32,7 +32,7 @@ export default function Home() {
         /**
          * @see https://docs.uploadthing.com/api-reference/react#uploadbutton
          */
-        endpoint="videoAndImage"
+        endpoint={(_) => _.videoAndImage}
```

Comment:
whatever we decide on, should probably be consistent. I think I prefer written out `routeRegistry`